resource/history/magento/project-community-edition/2.4.6-p8.json

```
@@ -0,0 +1,6 @@
+{
+    "require": {
+      "magento/composer-root-update-plugin": "~2.0",
+      "magento/composer-dependency-version-audit-plugin": "~0.1"
```

```suggestion
      "magento/composer-dependency-version-audit-plugin": "~0.1",
      "magento/composer-root-update-plugin": "~2.0"
```

do we want to keep alphabetical order for these entries so that's easier to compare?

---

I tend to keep the order used for the previous version 2.4.6-p7, since it's important to spot changes from one version to the next.

---

But since it's only two lines, I'm fine with the change.